### DISCUSSION TITLE
What happened to netplay with v3.1.6?

### DISCUSSION THREAD

**Discussion by Eriorry** [EmulatorJS/EmulatorJS#525] [General]
I made a website with a bunch of games that all had netplay on it. It worked fine a week ago but since a few days ago it started breaking. At first whenever i clicked the create room button it said the netplay was currently unavailable. Now the netplay button isnt even there anymore. What happened? Not even the netplay demo has it

**Comment by ethanaobrien** [comment]
I can no longer host a netplay server, due to lack of funds, and that I've been really tight on money recently. You must now host the netplay server manually. You can read how to [here](https://github.com/EmulatorJS/EmulatorJS-netplay)

**Reply by Eriorry** [reply]
how can i add it to my site? i used replit to make a netplay server

**Reply by allancoding** [reply]
Look at the docs: https://emulatorjs.org/docs4devs/Netplay.html

**Reply by Eriorry** [reply]
i added the url and the icon showed up, but it still says that netplay is currently unavailable

**Reply by ethanaobrien** [reply]
Again, read here: https://github.com/EmulatorJS/EmulatorJS-netplay/blob/main/README.md

This information is already provided
